docs/BuildConfiguration.md

```diff
@@ -35,7 +35,7 @@ Sections follow that describe particular settings in more depth.
 | maxTransferDuration                                                      |  `duration` | Transfers up to this duration with the default walk speed value will be pre-calculated and included in the Graph.                                              | *Optional* | `"PT30M"`                         |  2.1  |
 | [multiThreadElevationCalculations](#multiThreadElevationCalculations)    |  `boolean`  | Configuring multi-threading during elevation calculations.                                                                                                     | *Optional* | `false`                           |  2.0  |
 | [osmCacheDataInMem](#osmCacheDataInMem)                                  |  `boolean`  | If OSM data should be cached in memory during processing.                                                                                                      | *Optional* | `false`                           |  2.0  |
-| osmNaming                                                                |   `string`  | A custom OSM namer to use.                                                                                                                                     | *Optional* |                                   |  2.0  |
+| [osmNaming](#osmNaming)                                                  |    `enum`   | A custom OSM namer to use.                                                                                                                                     | *Optional* | `"default"`                       |  1.5  |
```

Comment:
Can you add the `sidewwalks` option in [this section of `doc-templates/BuildConfiguration.md`](https://github.com/opentripplanner/OpenTripPlanner/blob/2b2d36505ab952ffb6bcb2ec7c7ac84c4f609361/docs/BuildConfiguration.md#custom-naming)?

Reply:
Since it would be a duplication of the documentation I decided to remove the section that @binh-dam-ibigroup is referring to. All the possible values are already listed in this parameter table.

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/5774/commits/dbd8f2f1053c89ca183b5fa37e12612882394a70